llvm/unittests/Support/MathExtrasTest.cpp

```diff
@@ -183,15 +185,34 @@ TEST(MathExtras, NextPowerOf2) {
   EXPECT_EQ(256u, NextPowerOf2(128));
 }
 
-TEST(MathExtras, alignTo) {
+TEST(MathExtras, AlignTo) {
   EXPECT_EQ(8u, alignTo(5, 8));
   EXPECT_EQ(24u, alignTo(17, 8));
   EXPECT_EQ(0u, alignTo(~0LL, 8));
```

Comment:
this is somewhat a llvm style thing but do we prefer reinterpret/static_cast to c-style casts?

Reply:
We do, in general, but this particular case is just a simple cast from one integer type to another, preserving signedness, so I don't think it matters.

Reply:
Yes should be static_cast